portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBAdmin.testcase

```diff
@@ -95,6 +95,61 @@ definition {
 			kbArticleTitle = "Knowledge Base Article Title Edit");
 	}
 
+	@description = "This test covers LPS-171314. It ensures that the expiration label for Knowledge Base article works."
+	@priority = 4
+	test CanEnableArticleExpirationLabel {
+		property custom.properties = "company.default.time.zone=America/Los_Angeles${line.separator}feature.flag.LPS-165476=true";
+		property test.name.skip.portal.instance = "KBAdmin#CanEnableArticleExpirationLabel";
+
+		task ("Given the Notification Date is changed to 2 weeks") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Knowledge Base",
+				configurationName = "Service",
+				configurationScope = "System Scope");
```

Comment:
Can you create a new KBNavigator file with a macro that's similar to openToConfigInSystemSettings?